config/sync/block.block.featuredcontent.yml

```diff
@@ -18,6 +18,7 @@ settings:
   label: Featured
   provider: nidirect_homepage
   label_display: visible
+  featured_items: '14474'
```

Comment:
Could we automate the setting of this? Feels a bit risky to rely on us remembering to set it manually after each run (or have I misunderstood ?)

Reply:
Handled with env vars in https://github.com/dof-dss/nidirect-site-modules/pull/353